x/bank/spec/02_keepers.md

```diff
@@ -14,15 +14,15 @@ permissions are limited in the way that you expect.
 
 ## Blacklisting Addresses
 
-The `x/bank` module accepts a map of addresses that are considered blocklisted
+The `x/bank` module accepts a map of addresses that are considered blacklisted
```

Comment:
We need an alternative to "blacklist": https://developers.google.com/style/word-list#letter-b. This wording was intentional.

Reply:
Yeah, i was thinking about that when i was reading through the spec. How about `denylist`? I think i've seen a few org's make the change to `denylist` and `allowlist`.

Reply:
I don't have a strong preference personally

Reply:
Looks like blocklisted was already heavily used, just a few comments weren't changed. I'll go with blocklist